documentation/decisions/adr/2023-12-15-deliverable-reporting-strategy.md

```diff
@@ -93,18 +97,86 @@ We've decided to use the status of the deliverable in the product roadmap board
 
 ## Evaluation - Assigning issues to deliverables <!-- OPTIONAL -->
 
-### Milestone
+### Comparison matrix
 
-This option involves creating a milestone for each 30k deliverable and assigning issues to that milestone if they are required for delivery of that 30k.
+| Factor                                | Milestone per 10k | Milestone per 30k | Label | Deliverable column | Reserved phrase |
+| ------------------------------------- | :---------------: | :---------------: | :---: | :----------------: | :-------------: |
+| Supports filtering in repo            |        ❌         |        ✅          |  ✅   |         ❌         |        ❌        |
+| Supports filtering in GH project      |        ❌         |        ✅          |  ✅   |         ✅         |        ❌        |
+| Supports grouping in GH project       |        ❌         |        ✅          |  ❌   |         ✅         |        ❌        |
+| Supports grouping in GH reporting     |        ❌         |        ✅          |  ❌   |         ✅         |        ❌        |
+| Supports custom sorting in GH project |        ❌         |        ✅          |  ❌   |         ✅         |        ❌        |
+| Supports custom reporting             |        ✅         |        ✅          |  ✅   |         ✅         |        ✅        |
+| Enables engineers to organize issues  |        ✅         |        ❌          |  ✅   |         ✅         |        ✅        |
+| # of steps to create a deliverable    |         5         |        5          |   5   |         6          |        3        |
```

Comment:
I love this table! It's interesting we ended up choosing the option that takes the most steps for both creation and renaming, but I understand why, it has the most benefits.

Reply:
Hmm, the table says 6 steps to create a deliverable, but down below (under "Steps to create a 30k deliverable and assign an issue to it") it only shows 5? 

Edit: nevermind. I see that one of the 5 steps ("Add a new value for this 30k deliverable to the deliverable columns in:...") is actually 2 steps.

Reply:
Yeah I counted the numbered steps that had two sub-parts as 2 steps to be a bit more accurate. But agreed, it was helpful to compare them in this way!